server/dump.go

```diff
@@ -67,27 +73,15 @@ func (p *Plugin) createDump(ctx context.Context, min, max time.Time, remoteStora
 		return "", err
 	}
 
-	err = compressDirectory(dumpDir, zipFileName)
+	err = compressDirectory(dumpDir, tempZipFile)
 	if err != nil {
 		return "", err
 	}
-	defer os.Remove(zipFileName)
 
 	err = os.RemoveAll(dumpDir)
 	if err != nil {
 		return "", err
 	}
 
-	f, err := os.Open(zipFileName)
-	if err != nil {
-		return "", err
-	}
-	defer f.Close()
-
-	_, err = p.fileBackend.WriteFile(f, zipFileNameRemote)
-	if err != nil {
-		return "", err
-	}
-
-	return zipFileNameRemote, nil
```

Comment:
This is just shortcircuiting the final dump generation and only keeping it in localstore correct?

Reply:
Yep